code/controllers/configuration.dm

```diff
@@ -167,7 +167,7 @@ var/list/gamemode_cache = list()
 
 	var/simultaneous_pm_warning_timeout = 100
 
-	var/use_recursive_explosions = 0 //Defines whether the server uses recursive or circular explosions.
+	var/use_complex_explosions = 0 //Defines whether the server uses iterative or circular explosions.

```

Comment:
Consolidation of terms please. The system should either be referred to throughout the work as `complex_explosions` or as `iterative_explosions`. Unless we, for some reason, feel insane enough to have 3 types of explosions, with two of them as "complex".